tuf/client/updater.py

```diff
@@ -241,7 +244,7 @@ def __init__(self, updater_name):
     # Load pinned.json, which is required per TAP #4 and determines which
     # which targets should be sought from which repository(/ies).
     self._load_pinned_metadata(os.path.join(
```

Comment:
Note: In the pull request that implements TAP 4 (the map file), the path to the pinned file is given as an argument to MultiRepoUpdater().  See https://github.com/theupdateframework/tuf/pull/430#issue-207929275

We can decide on which to approach to use once we merge.